tests/functional/UseCase/Table/PreviewTableTest.php

```diff
@@ -51,6 +51,371 @@ protected function tearDown(): void
         $this->cleanTestProject();
     }
 
+    public function testArrayPreviewTable(): void
```

Comment:
nechtel bys tu metodu rozdelit bud do samostatnych testu nebo do metoda+provider? imho si to o to rika. ma to pres 1000 radku a asi 500 jsi pridal tedka ty.

Reply:
Je to tam prave takto schvalne aby ten test bol rychlejsi, vytvorim tablku, skontrolujem vsetky hodnoty ono sa totiz v setup metode setupuje novy projket vzdy 
https://github.com/keboola/php-storage-driver-bigquery/blob/e4eda8b4f635b5482c221bd21c3bd75cb148403e/tests/functional/UseCase/Table/PreviewTableTest.php#L37-L46

Reply:
👍 😢

Reply:
jeste na to koukam jedno a ...
1. to ze se vytvari projekt mi neprijde jako validni duvod, to se da udelat jako `setUpBeforeClass`
2. ten komentar je mimo, je to relevantni k `testPreviewTable`(tam je ten narust 500->1000) . imho se to z toho da docela vybruslit s nejakym polosigleton resenim kde tu tabulku vytvoris a ulozis do class property nebo tak neco a pak k tomu budes mit generator filtr+expectedData. Pak ve vystupu pekne uvidis, jaky usecase se testuje/pripadne padne. 

Jako jo, asi tady narazime na standardni "jirak to chce mit vyhezkane", ale za me 1000 radkova test metoda je too much. 

Davam approve i tak, toto necham na tobe